File: FinanceG5/src/main/java/com/lti/root/services/OrderDetailsService.java
package com.lti.root.services;

import java.util.List;

import com.lti.root.entity.CustomerDetails;
import com.lti.root.entity.OrderDetails;

public interface OrderDetailsService {
	
	boolean saveOrder(CustomerDetails o);
	List<OrderDetails> getAllOrders(int customerfk);
	
}

File: src/app/order-details.service.ts
import { HttpClient } from '@angular/common/http';
import { Injectable } from '@angular/core';
import { Observable } from 'rxjs';
import { CustomerDetails } from './customer-details';
import { OrderDetails } from './order-details';
import { ProductDetails } from './product-details';

@Injectable({
  providedIn: 'root'
})
export class OrderDetailsService {


  postOrder(customerupdated : CustomerDetails) {
    console.log(this.url+"/saveOrder",customerupdated)
    const body = JSON.stringify(customerupdated);
    console.log(body);
    this.http.post(this.url+"/saveOrder",customerupdated).subscribe(
      data=>{

      },error=>{
        console.log(error);
      }
    );
  }

  private url="http://localhost:8088";

  constructor(private http:HttpClient) { }

  
}
